lucasvdiepen/DerailedDeliveries
# Feature/camera-controller

All cameras are now cinemachine cameras.

**How to test:**
Go to develop scene and play normally. When you arrive at a station, switch to neutral gear. This will activate the "Enter" station camera transition. Increase your train gear to activate the "Exit" station camera transition.

You can obviously test this on any station. Train camera will now also shake a bit when going at its max speed (very subtle, feedback welcome) 

![stationCam](https://github.com/lucasvdiepen/DerailedDeliveries/assets/70685433/3cf6d6dd-c3d0-4ef6-8be4-3fd3a9cf8cde)

To test the local player tracker, enter the game and play normally. Make sure to parent the players to one of the wagons manually in the hierarchy (since the parenting features is still being made)
